src/Factory/ShopBillingDataFactoryInterface.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\RefundPlugin\Factory;
+
+use Sylius\Component\Resource\Factory\FactoryInterface;
+use Sylius\RefundPlugin\Entity\ShopBillingDataInterface;
+
+interface ShopBillingDataFactoryInterface extends FactoryInterface
+{
+    public function createNew(): ShopBillingDataInterface;
```

Comment:
I believe we don't have to define this method, as we extend the `FactoryInterface`

Reply:
i reverted it to decorating

Reply:
Still not needed, the method is in `FactoryInterface` nonetheless 😄

Reply:
yeah its not required to add it in this interface aswell